cirrus/server/glean/metrics.yaml

```diff
@@ -16,9 +16,6 @@ cirrus_events:
       branch:
         type: string
         description: The branch slug/identifier that was randomly chosen
-      enrollment_id:
-        type: string
-        description: A unique identifier generated at enrollment time
       experiment_type:
```

Comment:
you need to run the python glean commands to update the docs too,`python3 -m glean_parser translate -f markdown -o docs metrics.yaml pings.yaml
`

Reply:
AH great callout, thank you